client/src/partials/fiscal/templates/modals/fiscal.closing.modal.html

```diff
@@ -0,0 +1,111 @@
+              <dd>
+                <div class="text-right">
+                  {{ $ctrl.charge | currency }}
+                </div>
+              </dd>
+            </h3>
+
+            <!-- results  -->
+            <h3 style="border-top: 1px dashed #ccc; margin-top: 10px;" class="text-bold">
+              <dt>{{ 'FORM.LABELS.RESULT' | translate }}</dt>
+              <dd>
+                <div class="text-right" ng-class="{'text-danger' : $ctrl.globalResult < 0, 'text-success' : $ctrl.globalResult > 0 }">
+                  {{ $ctrl.globalResult | currency }}
+                </div>
+              </dd>
+            </h3>
+          </dl>
+
+          <br>
+
+          <div class="form-group" ng-class="{ 'has-error' : ClosingFiscalYear.$submitted && ClosingFiscalYear.resultAccount.$invalid }">
+            <label class="control-label">
+              {{ "FORM.LABELS.RESULT_ACCOUNT_SCT" | translate }}
+            </label>
+
+            <ui-select name="resultAccount" ng-model="$ctrl.resultAccount" theme="bootstrap" required>
+              <ui-select-match placeholder="{{ 'FORM.PLACEHOLDERS.ACCOUNT' | translate }}">
+                <strong>{{$select.selected.number}}</strong> <span>{{$select.selected.label}}</span>
+              </ui-select-match>
+              <ui-select-choices ui-select-focus-patch repeat="account in $ctrl.accounts | filter:$select.search">
+                <strong ng-bind-html="account.number | highlight:$select.search"></strong>
+                <span ng-bind-html="account.label | highlight:$select.search"></span>
+              </ui-select-choices>
+            </ui-select>
+
+            <div class="help-block" ng-messages="ClosingFiscalYear.resultAccount.$error" ng-show="ClosingFiscalYear.$submitted">
+              <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+            </div>
+          </div>
+
+        </div>
+
+        <div class="text-center" ng-switch-when="summary">
+          <h3>{{ 'FISCAL.SOLD_DESCRIPTION' | translate }}</h3>
+          <h3 class="text-primary">{{ $ctrl.resultAccount.number + ' - ' + $ctrl.resultAccount.label }}</h3>
+          <h3>
+            {{ 'FISCAL.RESULT_AMOUNT' | translate }}:
+            <span ng-class="{'text-danger' : $ctrl.globalResult < 0, 'text-success' : $ctrl.globalResult > 0 }">
+              {{ $ctrl.globalResult | currency }}
+            </span>
+          </h3>
+        </div>
+
+      </div>
+
+    </div>
+
+    <div ng-show="$ctrl.fiscal.locked">
+      <h1 class="text-center">
+        <strong>{{ $ctrl.fiscal.label }}</strong><br>
+        {{ 'FISCAL.ALREADY_CLOSED' | translate }}
+      </h1>
+    </div>
+
+  </div>
+
+  <div class="modal-footer">
+    <!-- cancel  -->
+    <button type="button" class="btn btn-default" ng-click="$ctrl.cancel()">
+      {{ 'FORM.BUTTONS.CANCEL' | translate }}
+    </button>
+
+    <!-- submit  -->
+    <button type="submit" class="btn btn-primary" ng-disabled="$ctrl.fiscal.locked">
```

Comment:
Because this button will do an async operation at the end, I propose you use a `bhLoadingButton`.  It should work the same way.